content/en/docs/cve.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Common Vulnerabilities and Exposures
+weight: 102
+---
+
+This is a list of reported CVEs across all repositories in the OpenTelemetry org
+on GitHub. The raw data is stored in
+[this GitHub repository](https://github.com/open-telemetry/sig-security), and it
+is refreshed daily.
+
+  <table id="cve-table">
+    <thead>
+      <tr>
+        <th>CVE ID</th>
+        <th>Issue Summary</th>
+        <th>Severity</th>
+        <th>Repository</th>
+      </tr>
+    </thead>
+    <tbody>
+    </tbody>
+  </table>
+
+<script id="main-script">
```

Comment:
I guess, the purpose of this script is to make sure that the most up-to-date data is always live on the website? 

There is an alternative way: hugo supports the ability to read from JSON files (https://gohugo.io/functions/data/getjson/), so we could replace the javascript with a template function and the reload with a re-build of the website triggered on a new CVE being published? The **advantage** would be that we don't need javascript, which feels a little bit more accessible and also makes it easier for crawlers to collect this page (helping the CVEs to be findable)

We can start with the JS, but I would like to get @chalin's perspective when he's back

Reply:
Yes, let's do build-time fetch of the data if y'all agree!

Reply:
How often is the site rebuilt? If we do this, should I remove the text about the list being updated daily?

Reply:
As @svrnm suggested, let's go with the dynamic update for now. We can revisit the page in 2024 :).